### DISCUSSION TITLE
Access $variant in variables.scss

### DISCUSSION THREAD

**Discussion by milaan-muc** [inkline/inkline#225] [Need help] [answered]
Is there a way to access the current variant like dark and light in variables.scss?

**Comment by alexgrozav** [comment, marked as answer]
Hey! If you're referring to the current global color mode (light or dark), it can be determined when styling using either:

- the `.inkline.-light` and `.inkline.-dark` classes that apply to the document body
- the `@media (prefers-color-scheme: dark)` media query

However, this won't really help you with the `variables.scss` file.

Is this what you're looking for? If not, can you please give me an example use case?

**Reply by milaan-muc** [reply]
I have to admit im a scss newbie :-)

I want to set new values for `$color-primary` in the `variables.scss` file and i was wondering how can specify the value depending on dark/light mode.

Can this be achived by this in the `variables.scss` file?

```
.inkline { 
    &.-light { $color-primary: #ff0000; }
    &.-dark { $color-primary: #00ff00; }
```

**Reply by alexgrozav** [reply]
I have an update on this. It will be easily doable in Inkline 3. Stay tuned! :)
